src/seedu/addressbook/data/person/Person.java

```diff
@@ -14,8 +14,11 @@
     private Phone phone;
     private Email email;
     private Address address;
+    private int seqNumber;
+    private static int nextSeqNumber = 1;
```

Comment:
This is not the correct naming convention for a constant value (all upper cases with '_')